site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.stories.tsx

```diff
@@ -86,6 +86,13 @@ export const Parameters: Story = {
         ephemeral: false,
       },
     ],
+    autofillParameters: [
+      {
+        name: "first_parameter",
+        value: "It works!",
+        source: "user_history",
```

Comment:
Why would that be better?

Reply:
```
      {
        name: "first_parameter",
        value: "It works!",
        source: "user_history",
      }
```

Again, readability. When you see a block like the above one, it can be misleading that the source can take any string.